Header.c

```diff
@@ -121,6 +121,14 @@ void Header_populateFromSettings(Header* this) {
    const Settings* settings = this->host->settings;
    Header_setLayout(this, settings->hLayout);
 
+   int num = load_flex_modules();
+   int platform_size = 0;
+
+   for (platform_size = 0; Platform_meterTypes[platform_size] != NULL; platform_size++);
+   for (int i = 0; i < num; i++) Platform_meterTypes[platform_size+i]=FlexMeter_class+i;
+
+   Platform_meterTypes[platform_size+num]=NULL;
```

Comment:
Space around binary operators:

```suggestion
   for (platform_size = 0; Platform_meterTypes[platform_size] != NULL; platform_size++) {
      // Count meter types for the platform
   }
   for (int i = 0; i < num; i++) {
      Platform_meterTypes[platform_size + i] = FlexMeter_class + i;
   }

   Platform_meterTypes[platform_size + num] = NULL;
```

Furthermore, `for` loops always should have a body; and be it empty.